Ratings_MVP/Ratings_MVP/Helper/RouterProtocol.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  RouterProtocol.swift
+//  Ratings_MVP
+//
+//  Created by Валентин Коскин on 10.01.2023.
+//
+
+import UIKit
+
+/// Router protocol
+protocol RouterProtocol: RouterMainProtocol {
```

Comment:
```suggestion
protocol RouterProtocol: RouterMainProtocol {
```
Переименовала бы протокол и сам класс, если у тебя вдруг будет еще один экран и еще один роутер, то его тоже нудно будет как-то называть)